File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1114_basico.js
import { question } from "readline-sync";

function main() {
    let senha_correta = `2002`;

    while (true) {
        let senha = get_text();

        if (senha === senha_correta) {
            print(`Acesso Permitido`);
            break;
        } else {
            print(`Senha Invalida`);
        }
    }
}

function get_text() {
    return question();
}

function print(texto) {
    return console.log(texto);
}

main();

File: Exercicios_ED_I/Bee/1114_basico.cpp
#include <stdio.h>
#include <iostream>
using namespace std;

main() {
    int senha;
    cin >> senha;
    while (senha != 2002) {
        cout << "Senha Invalida" << endl;
        cin >> senha;
    }
    cout << "Acesso Permitido" << endl;
}